### PR TITLE
Feature: Inventory Focus Mode

### PR DESCRIPTION
**Pull request by Thunderblade73**
## What
Only show the name of the item when in focus mode.

## Discord
https://discord.com/channels/997079228510117908/1292289101340344453

## Changelog New Features
+ Added Focus Mode. - Thunderblade73
    * In Focus Mode, only the name of the item is displayed instead of the full description.

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/config/features/inventory/FocusModeConfig.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+import org.lwjgl.input.Keyboard;
+
+public class FocusModeConfig {
+
+    @Expose
+    @ConfigOption(name = "Enable", desc = "In focus mode you only see the name of the item instead of the whole description.")
+    @ConfigEditorBoolean
+    @FeatureToggle
+    public boolean enable = false;
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/config/features/inventory/FocusModeConfig.java]: Both of these should be enabled not enable

File: src/main/java/at/hannibal2/skyhanni/config/features/inventory/InventoryConfig.java [hunk 2]
```diff
@@ -84,6 +84,11 @@ public class InventoryConfig {
     @Accordion
     public PersonalCompactorConfig personalCompactor = new PersonalCompactorConfig();
 
+    @Expose
+    @ConfigOption(name = "Focus Mode",desc="")
```
Comment (jani270) [inline src/main/java/at/hannibal2/skyhanni/config/features/inventory/InventoryConfig.java]: 🤓 
```suggestion
    @ConfigOption(name = "Focus Mode", desc="")
```

Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/config/features/inventory/InventoryConfig.java]:
```suggestion
    @ConfigOption(name = "Focus Mode", desc="")
```

**Review by CalMWolfs — CHANGES_REQUESTED** [review]

File: src/main/java/at/hannibal2/skyhanni/config/features/inventory/InventoryConfig.java (hunk 2 above)
Comment (jani270) [reply src/main/java/at/hannibal2/skyhanni/config/features/inventory/InventoryConfig.java]: late

Comment (CalMWolfs) [reply src/main/java/at/hannibal2/skyhanni/config/features/inventory/InventoryConfig.java]: > late

😭

**Review by CalMWolfs — APPROVED** [review]
i was very focussed while using this

**Pull request merged**